Hollowborn/HollowbornChaosEnvoy/CoreHollowbornChaosEnvoy.cs

```diff
@@ -253,6 +264,33 @@ public void PersistingMayhem(bool getAll = true)
             foreach (string s in rewards)
                 Bot.Wait.ForPickup(s);
         }
+        Core.ToBank(rewards);
         Core.CancelRegisteredQuests();
     }
+    
+    /// <summary>
+    /// Joins a map and aggro, then kill the monsters
+    /// Do not aggro too many monsters
+    /// </summary>
+    /// <param name="map">Map to join</param>
+    /// <param name="monsterNames">Name of the monsters to kill. Recommended to pick monster with highest quantity and lowest stats</param>
+    /// <param name="cell">Cell to jump to. Recommended to pick cell with most monsters</param>
+    /// <param name="item">Item to hunt the monster for, if null will just hunt & kill the monster 1 time</param>
+    /// <param name="quant">Desired quantity of the item</param>
+    /// <param name="isTemp">Whether the item is temporary</param>
+    /// <param name="pad">Pad to jump to. Set default to Spawn because needed to bypass the Army restriction</param>
+    public void AggroKill(string map, string [] monsterNames, string cell, string item, int quant, bool isTemp = true, string pad = "Spawn")
+    {
+        if (item != null && (isTemp ? Bot.TempInv.Contains(item, quant) : Core.CheckInventory(item, quant)))
+            return;
+        if (!isTemp && item != null)
+            Core.AddDrop(item);
+        
+        Core.Join(map, cell, pad);
+        Army.AggroMonNames(monsterNames);
+        Army.AggroMonStart(map);
+        while (!Bot.ShouldExit && (isTemp ? !Bot.TempInv.Contains(item, quant) : !Core.CheckInventory(item, quant)))
+            Bot.Combat.Attack("*");
+        Army.AggroMonStop(true);
+    }
```

Comment:
Denied, as a standalone bot it's not intended to be using aggro methods, people without army would get in trouble for this. Most of our user base doesn't use army